services/boxHistoryService.js

```diff
@@ -17,15 +17,33 @@ const getTransactionByID = async (transactionID) => {
 const getBoxesWithStatusOrPickup = async (status, pickup) => {
   let res = null;
   try {
-    res = await db.query(
-      `SELECT * FROM "Box_History"
-      WHERE
-        ($(status) = '' OR status = $(status))
-        ${pickup ? 'AND pickup = $(pickup)' : ''}
-      ORDER BY
-        pickup, box_id;`,
-      { status, pickup },
-    );
+    if (status === 'evaluated') {
+      res = await db.query(
```

Comment:
Do you know of a more efficient solution to do this? I'm not able to find another way to find the most recently submitted distinct boxes. The box approval process already takes a while to load data as it is so it's a bit hard to determine how long this query is taking vs what it already is by default. It might be better to work on this after pagination is complete so we know if this is an issue or not. Let me know what you think.

Reply:
Okay I'll look into timing this later tonight

Reply:
554ms lfg